frontend/src/components/interface/appBar/AppBar.vue

```diff
@@ -28,10 +30,49 @@
             },
         },
 
+        data() {
+            return {
+                loggedIn: false,
+            };
+        },
+        mounted() {
+            this.verify();
+        },
+
         methods: {
             redirectToRoot() {
                 redirectTo(window.location.origin + window.location.pathname);
             },
+
+            login() {
```

Comment:
Wszystkie te funkcje przeniósłbym do osobnego store i tam przechowywał to info o userze, ale to jak nie chcesz się bawić na frontendzie, to można to oddelegować do innego zadania.